README.md

```diff
@@ -304,6 +306,15 @@ The `copy` command only works for instances that have been created by lima 0.5.0
 < ~/.lima/_config/user.pub limactl shell INSTANCE sh -c 'tee -a ~/.ssh/authorized_keys'
 ```
 
+### Networking
+#### "Cannot access the guest IP 192.168.5.15 from the host"
+
+The default guest IP 192.168.5.15 is not accessible from the host and other guests.
+
+To allow accessing the guest IP from the host and other virtual machines, enable [`vde_vmnet`](https://github.com/lima-vm/vde_vmnet).
```

Comment:
Maybe reword to make clear that this will add one or more additional routable IPs to the guest. It will still not allow access "the" guest IP.

Reply:
updated